module/skill/SplineKick/data/config/SplineKick.yaml

```diff
@@ -0,0 +1,38 @@
+# Controls the minimum log level that NUClear log will display
```

Comment:
is this tuned for webots? do you wanna put it in a webots folder and just duplicate it for the default config so when we tune it for the real robot we don't lose the webots tune. If you remember some of the values we did the other day could add that in here. Afaik we made the time longer and made the torso move across less.